src/jit/codegencommon.cpp

```diff
@@ -3240,7 +3240,11 @@ void CodeGen::genGenerateCode(void** codePtr, ULONG* nativeSizeOfCode)
     }
 #endif // EMIT_TRACK_STACK_DEPTH
 
-    *nativeSizeOfCode                 = codeSize;
+    if (nativeSizeOfCode != nullptr)
+    {
+        *nativeSizeOfCode = codeSize;
+    }
+
```

Comment:
@noahfalk 
I set this checking when I didn't know well coreclr just in case. No didn't see any code path with null nativeSizeOfCode